code/modules/awaymissions/mission_code/ghost_role_spawners/golems.dm

```diff
@@ -134,6 +134,24 @@
 		user.death()
 		return
 
+/obj/effect/mob_spawn/human/golem/attackby(obj/item/slimepotion/transference/I, mob/user, params)
+	. = ..()
+	if(istype(src, /obj/effect/mob_spawn/human/golem/servant))
```

Comment:
It's more standard to keep the argument the same and typecast it later, since that'll help avoid any issues.
```suggestion
/obj/effect/mob_spawn/human/golem/attackby(obj/item/I, mob/user, params)
	. = ..()
	if(!istype(I, /obj/item/slimepotion/transference))
		return
	if(istype(src, /obj/effect/mob_spawn/human/golem/servant))
```